src/vfxt/src/createUiDefinition.json

```diff
@@ -77,7 +77,7 @@
           "type": "Microsoft.Common.TextBox",
           "label": "Avere cluster create role ID",
           "defaultValue": "8e3af657-a8ff-443c-a75c-2fe8c4bcb635",
-          "toolTip": "The AAD role id used for creation of the vFXT cluster.  The default is the 'Owner' role.  However, we recommend you create your own role as described in the managed identity documentation, and you obtain the GUID by using the following az command: az role definition list --query '[*].{roleName:roleName, name:name}' -o table --name 'Avere Contributor'.",
+          "toolTip": "The RBAC role ID used to create the vFXT cluster. The default is the 'Owner' role, but you should create your own role as described in the managed identity documentation. Obtain the custom role's GUID with the following az command: az role definition list --query '[*].{roleName:roleName, name:name}' -o table --name 'Avere Contributor'.",
```

Comment:
As of today (1/16), this is correct. It might not be correct by Monday. We may be able to remove this field.